docs/internationalization_translation.md

```diff
@@ -39,3 +39,13 @@ Native Name: Deutsch
 2. Add the language tag and native name in `src/translations/translationMetadata.json`.  Examples: "Français", "Français (CA)"
 3. Add the new language in Lokalize.
 Note: Sometimes you have to change the tag in Lokalise (Language -> Language settings -> custom ISO code).
+
+## Language specific guidelines
+Most languages have multiple possibilities of translating a sentence. Please take a look at the guidelines for your language here, where you can also find some typical mistakes to prevent.
```

Comment:
```suggestion
Most languages have multiple possible of translations of a sentence. Please take a look at the guidelines for your language here, where you can also find some typical mistakes to prevent.
```